sbin/jtreg_prep.sh

```diff
@@ -69,24 +69,24 @@ fi
 
 # Step 1: Fetch OpenJDK, as that's where the tests live.
 
-if [ -d "$WORKING_DIR"/$OPENJDK_REPO_NAME/.git ] && [ $REPOSITORY == "AdoptOpenJDK/openjdk-jdk8u" ] ; then
+if [ -d "$WORKING_DIR"/$OPENJDK_REPO_NAME/.git ] && [ "$REPOSITORY" == "AdoptOpenJDK/openjdk-jdk8u" ] ; then
   # It does exist and it's a repo other than the AdoptOpenJDK one
-  cd $WORKING_DIR/$OPENJDK_REPO_NAME
+  cd "$WORKING_DIR/$OPENJDK_REPO_NAME"  || exit
   echo "Will reset the repository at $PWD in 10 seconds..."
   sleep 10
   echo "Pulling latest changes from git repo"
   git fetch --all
   git reset --hard origin/$BRANCH
-  cd $WORKING_DIR
+  cd "$WORKING_DIR" || exit
 elif [ ! -d "${WORKING_DIR}"/$OPENJDK_REPO_NAME/.git ] ; then
   # If it doesn't exist, clone it
   echo "Didn't find any existing openjdk repository at WORKING_DIR (set to ${WORKING_DIR}) so cloning the source to openjdk"
   if [[ "${USE_SSH}" == true ]] ; then
     echo "git clone -b ${BRANCH} git@github.com:${REPOSITORY}.git $WORKING_DIR/$OPENJDK_REPO_NAME"
-    git clone -b ${BRANCH} git@github.com:${REPOSITORY}.git $WORKING_DIR/$OPENJDK_REPO_NAME
+    git clone -b ${BRANCH} git@github.com:"${REPOSITORY}".git "$WORKING_DIR/$OPENJDK_REPO_NAME"
```

Comment:
As above, the dirs definitely need quoting to get through to the command.
Will add {} for house style.